### PR TITLE
Drop support of IE11

### PR DESCRIPTION
**Pull request by juergba**
### Description

We drop support of Internet Explorer 11.

- **mocha.js**: browser bundle transpiled to **ES5** is dropped
- **mocha-es2018.js**: browser bundle in **ES2018** is renamed to **mocha.js**

### Description of the Change

| devDependencies | from | to |  |
|------------|----------|----------|----------|
| @babel/preset-env | removed |
| @rollup/plugin-babel | removed |
| core-js | removed |
| regenerator-runtime | removed |
| rollup | 2.66.0 | 2.70.1 |
| @rollup/plugin-commonjs | 14.0.0 | 21.0.2 |
| @rollup/plugin-multi-entry | 3.0.1 | 4.0.1 |
| @rollup/plugin-node-resolve | 8.4.0 | 13.1.3 |
| rollup-plugin-visualizer | 5.5.4 | 5.6.0 |
| rollup-plugin-node-polyfills | replaced | | rollup-plugin-polyfill-node |

### CODE REVIEW

**Comment by juergba** [comment]
@giltayar do we still need to create/publish a sourcemap of our ES2018 browser bundle?
We don't transpile anymore, so I think we don't, right?

**Comment by giltayar** [comment]
@juergba while we don't transpile anymore, we still bundle, so theoretically a source map is still needed.

**Comment by juergba** [comment]
@giltayar ok, thanks. I will keep the source map then.

File: karma.conf.js [hunk 1]
```diff
@@ -29,15 +29,9 @@ const BASE_BUNDLE_DIR_PATH = path.join(__dirname, '.karma');
...
-  fs.renameSync('./mocha.js', './mocha-es2018.js');
-  fs.renameSync('./mocha-es5.js', './mocha.js');
-}
-
 const SAUCE_BROWSER_PLATFORM_MAP = {
   'chrome@latest': 'Windows 10',
   'MicrosoftEdge@latest': 'Windows 10',
-  'internet explorer@latest': 'Windows 10',
   'firefox@latest': 'Windows 10',
   'safari@latest': 'macOS 10.13'
```
Comment (juergba) [inline karma.conf.js:36]: @giltayar @outsideris  which macOS do I configure here? 10.15 Catalina / 11 Big Sur / 12 Monterey ?

**Pull request merged**